packages/foam-vscode/src/features/utility-commands.ts

```diff
@@ -27,7 +22,7 @@ const feature: FoamFeature = {
       vscode.commands.registerCommand(
         OPEN_COMMAND.command,
         async (params: { uri: URI }) => {
-          const { uri } = params;
+          const uri = new URI(params.uri);
```

Comment:
minor - why are we creating a new object here?

Reply:
Because what is coming is a parse of the json encoded object, which is just a bare object, not a proper instance of `URI`. So object-things do work but instance-things don't. I can't remember which was the problem right now, but I guess that `uri.getName()` was.